muxinc/elements
# feat(mux-player): add storyboard-src attribute

## Description

This PR adds the `storyboard-src` attribute to `<mux-player>`. This attribute should allow users to directly set the source URL from the storyboard rather than relying on the player to infer it from the `playbackId`.

This PR also adds `storyboardSrc` as a prop for `<MuxPlayer>`.

## Motivation

At times it's necessary to derive the storyboard source URL from something other than the `playbackId`.

## Steps to test
- In `examples/nextjs-with-typescript/`, run `yarn dev`
  - navigate to the `<MuxPlayer>` page
  - select the `Landscape VOD with storyboard-src` asset
  - edit the `storyboardSrc` prop
- Run the `should work with storyboard-src'` unit test
